src/main/kotlin/org/rust/ide/annotator/RsHighlightingMutableAnnotator.kt

```diff
@@ -66,16 +59,16 @@ class RsHighlightingMutableAnnotator : Annotator {
     }
 
     private fun addHighlightingAnnotation(holder: AnnotationHolder, target: PsiElement, key: RsColor) {
-        // These following lines allow to test the mutable highlight for it own using INFORMATION and the description.
-        val annotationSeverity = if (isUnitTestMode) HighlightSeverity.INFORMATION else MUTABLE_HIGHTLIGHTING
-        val annotationText = if (isUnitTestMode) key.attributesDescriptor.displayName else null
-        holder.createAnnotation(
-            annotationSeverity,
-            target.textRange,
-            annotationText
-        ).textAttributes = key.textAttributesKey
+        val annotationSeverity = if (isUnitTestMode) key.testSeverity else MUTABLE_HIGHLIGHTING
+        holder.createAnnotation(annotationSeverity, target.textRange, null).textAttributes = key.textAttributesKey
     }
 
+    companion object {
+        private val MUTABLE_HIGHLIGHTING = HighlightSeverity(
+            "MUTABLE_HIGHLIGHTING",
+            HighlightSeverity.INFORMATION.myVal + 3
```

Comment:
Why 3?

Reply:
Just some positive value to be greater than `HighlightSeverity.INFORMATION`
If I remember correctly it was inspired by [LOW_PRIORITY_HIGHLIGHTING](https://github.com/JetBrains/intellij-community/blob/master/python/src/com/jetbrains/python/validation/HighlightingAnnotator.java#L37) highlight severity in python plugin